doc/udffsck.8

```diff
@@ -0,0 +1,162 @@
+.\" Copyright 2017 Vojtech Vladyka <vojtech.vladyka@gmail.com>
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
+.\" the License, or (at your option) any later version.
+.\"
+.\" The GNU General Public License's references to "object code"
+.\" and "executables" are to be interpreted as the output of any
+.\" document formatting or typesetting system, including
+.\" intermediate and printed output.
+.\"
+.\" This manual is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\" GNU General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU General Public
+.\" License along with this manual; if not, write to the Free
+.\" Software Foundation, Inc., 51 Franklin Street, Fifth Floor, 
+.\" Boston, MA 02110-1301 USA.
+.\"
+.\" References consulted:
+.\"   
+.\"
+.\"
+.TH UDFFSCK 8 "May 2017" "UDFTOOLS 1.3"
```

Comment:
Remove version number from man page to prevent version number out of date (e.g like now).

Reply:
Reused header from mkudffs, commit here:  983f572